tests/integration/test_api_gateway.py

```diff
@@ -577,3 +577,116 @@ def create_lambda_function(fn_name):
             func_name=fn_name,
             runtime=LAMBDA_RUNTIME_PYTHON27
         )
+
+    def test_create_model(self):
+        client = aws_stack.connect_to_service('apigateway')
+        response = client.create_rest_api(name='my_api', description='this is my api')
+        rest_api_id = response['id']
+        dummy_rest_api_id = 'a12b3c4d'
```

Comment:
nit: Perhaps this ID could be set to something like this, to make it clear that we're simulating a non-existing ID here (and not a hard-coded ID):
```
dummy_rest_api_id = '_non_existing_'
```